pkg/operation/botanist/component/kubeapiserver/kube_apiserver.go

```diff
@@ -211,6 +213,8 @@ type Images struct {
 	KubeAPIServer string
 	// VPNSeed is the container image for the vpn-seed.
 	VPNSeed string
+	// VPNClient is the container image for the vpn-seed-client.
```

Comment:
But we inject the `vpn-shoot-client` image. Confusion...

Reply:
Well, that's the current name of the image.